README.md

```diff
@@ -49,14 +47,10 @@ Follow these steps in order to install and enable communitheme.
 2. Restart your computer. The login screen will now use Communitheme by default. Click on your user, click on the gear icon ans select the "Ubuntu with communitheme snap" session from the login screen, and login.
 3. Now everything is using the communitheme including applications, icons, sound notifications and cursor. Any new update of communitheme will come directly to you on a regular basis thanks to the snap without having to run any command!
 
-The following video guides you through these steps.
-
-[![Video guide](https://img.youtube.com/vi/azlreXxAigY/0.jpg)](https://www.youtube.com/watch?v=azlreXxAigY)
```

Comment:
It seems to me that the link is still valid here https://www.youtube.com/watch?v=azlreXxAigY

Reply:
Uh strange.... : 

![image](https://user-images.githubusercontent.com/15329494/57183327-95035f80-6eab-11e9-861f-d7a8d01c607c.png)

Reply:
Totally :/

![image](https://user-images.githubusercontent.com/2883614/57191402-6de97400-6f25-11e9-9e20-b8b9cb6fab9a.png)

Reply:
Could be something for germany only? 
Would it also be okay if I attach a GIF instead?

Reply:
No idea, but the GIF would be fine